static/js/clip.js

```diff
@@ -6,12 +6,16 @@
 (function(document, Clipboard) {
 
     var $codes = document.querySelectorAll('pre');
+    const codelangtext = document.querySelectorAll('.language-text');
 
     function addCopy(element) {
```

Comment:
I think you're on the right path with this. What I'd suggest is that instead of searching for the text elements before we add the button, we instead search for the `.language-text` class within the selected pre tags, and then if we get an hits simply don't add the copy button. (This pattern is often called a guard statement, as we're _guarding_ the `addCopy` function from proceeding when one or more conditions are met.)

```suggestion
    function addCopy(element) {
      const codelangtext = element.querySelectorAll('.language-text');
      if (codelangtext.length) {
        return;
      }
```